### ISSUE TITLE
Activities access to NUTS level 3 spatial polygons

### ISSUE THREAD

**Issue by robin-hutchinson** [BiologicalRecordsCentre/iRecord#1769]
We've had a request to be able to filter to the modern boundary for Oxfordshire on the activities page. Usually I would use BC boundaries or LERC boundaries but neither cover just Oxfordshire. I do have one in the NUTS level 3 list I can use, but that category isn't available in the activities pages.

Would it be possible to add this?

**Comment by johnvanbreda** [comment]
Yes, NUTS level 3 is available as an indexed layer so I can add it. However should we check with @kitenetter first in case there are support implications, as there will be a huge list of NUTS 3 boundaries available to select from, most of which are not UK? Also we might want to consider including any additional layers on the Explore pages so they are consistent?

**Comment by robin-hutchinson** [comment]
That makes sense thank you - happy to leave this to the new year to figure out the best solution!
